components/exercises/chapter-3-oscar/BeanCounting.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+
+export function countBs(param: string): number {
```

Comment:
Add docs for each function, you can use github copilot /doc

Reply:
docs added